server.js

```diff
@@ -14,118 +18,227 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 // Serve client from build folder
 app.use(express.static(path.join(__dirname, "build")));
-
+// Enable cors
+app.use(cors());
 // enables environment variables by
 // parsing the .env file and assigning it to process.env
 dotenv.config({
   path: "./.env",
 });
 
+const isMock = process.env.REACT_APP_IS_MOCK === "1";
+const SANDBOX_URL = process.env.REACT_APP_SANDBOX_URL;
```

Comment:
@longxice inconsistent style btw constants, e.g: SANDBOX_URL vs isMock
@leonthaibao what is your suggestion?

Reply:
@anhldbk For consistency, I will change it, from isMock to IS_MOCK